e2e-polybft/cardanofw/test-apex-system.go

```diff
@@ -330,6 +415,51 @@ func (a *ApexSystem) GenerateConfigs() error {
 	})
 }
 
+func (a *ApexSystem) GenerateSkylineConfigs() error {
```

Comment:
No need for two public functions. There should be two private functions and one existing public function that calls those two in an if-else statement, depending on some config (`IsReactor` or `IsSkyline` if that is not already in the config).